client/actions/test/editActions.test.js

```diff
@@ -19,11 +19,18 @@ describe('editActions', () => {
     expect(changeSavedStatus.payload.isSaved).toBe(true)
   })
 
-  it('#changeSection sets the activeSection to arg', () => {
-    const changeSection = editActions.changeSection('display')
+  it('#changeSection sets activeSection to arg', () => {
+    const changeSection = editActions.changeSection(6)
```

Comment:
Wondering -- is there any way to avoid numeric indexes for stuff like this? They don't convey much information to the outside world and tend to be brittle.

Reply:
Planning to move the sections collection into the Redux state eventually, where potentially this could use the section object itself over an index. However that requires changing many components to support, so this is an incremental solution that sticks to our current system of finding a section by index.

Reply:
Sounds good 👍